lib/pos_blockchain.go

```diff
@@ -646,6 +667,10 @@ func (bc *Blockchain) upsertBlockAndBlockNodeToDB(block *MsgDeSoBlock, blockNode
 			}
 		}
 
+		// TODO: if storeFullBlock = false, then we should probably remove the block from the DB? This can
+		// happen if we had a block stored in the DB but then determined that it would have failed validation.
+		// We would need to evict the block from the DB in that case.
+
```

Comment:
I think this is a later optimization we can make. I *think* it's fine, but I'd rather deal with this performance enhancement later.

Reply:
Makes sense. Definitely a disk space optimization for the future